pkg/registry/networking/servicecidr/strategy_test.go

```diff
@@ -15,3 +15,84 @@ limitations under the License.
 */
 
 package servicecidr
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/kubernetes/pkg/apis/networking"
+)
+
+func newServiceCIDR() *networking.ServiceCIDR {
+	return &networking.ServiceCIDR{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            "servicecidr-test",
+			ResourceVersion: "1",
+		},
+		Spec: networking.ServiceCIDRSpec{
+			CIDRs: []string{"10.10.0.0/24"},
+		},
+	}
+}
+
+func TestServiceCIDRStrategy(t *testing.T) {
+	if Strategy.NamespaceScoped() {
+		t.Errorf("Expected ServiceCIDR to be cluster-scoped")
+	}
+
+	resetFields := Strategy.GetResetFields()
+	if len(resetFields) != 1 {
+		t.Errorf("ResetFields should have 1 element, but have %d", len(resetFields))
+	}
+	obj := &networking.ServiceCIDR{Spec: networking.ServiceCIDRSpec{CIDRs: []string{"bad cidr"}}}
+
+	errors := Strategy.Validate(context.TODO(), obj)
+	if len(errors) == 0 {
+		t.Errorf("Expected validation errors for invalid object")
+	}
```

Comment:
same comment as before, assert on the exact number of errors we are expecting

Reply:
Done, thank you very much.